src/rules/quotemarkRule.ts

```diff
@@ -75,18 +73,12 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 
     public isEnabled(): boolean {
-        return super.isEnabled() && (this.ruleArguments[0] === OPTION_SINGLE || this.ruleArguments[0] === OPTION_DOUBLE);
+        return super.isEnabled() && hasSingleDoublePreference(this.ruleArguments);
```

Comment:
If we remove this method, the rule will default to double quotes? I think that's more reasonable than just doing nothing.